test/ranking.spec.js

```diff
@@ -0,0 +1,112 @@
+import {
+    Ranking
+} from '../src/app/Ranking';
+
+describe("Ranking's logic", () => {
+    const peopleRanking = new Ranking('people');
+
+    const results = [{
+        'user': 'user1',
+        'score': 15,
+        'maxScore': 30 // 50%
+    }, {
+        'user': 'user2',
+        'score': 18,
+        'maxScore': 40 // 45%
+    }, {
+        'user': 'user3',
+        'score': 15,
+        'maxScore': 25 // 60%
+    }];
+
+    afterEach(() => {
+        localStorage.clear();
+    });
+
+    it('When set wrong category name, exception will be thrown', () => {
+        expect(() => {
+            new Ranking('vehicle');
+        }).toThrow('"vehicle" mode is not valid!');
+    });
+
+    it('When new score is higher than or equal to at least one of top results, return true', () => {
+        localStorage.setItem('people', JSON.stringify(results));
+        const score = 9;
+        const maxScore = 18; // 45%
+
+        expect(peopleRanking._isHigherThanCurrentTop(score, maxScore)).toBeTruthy;
+    });
+
+    it('When new score is less than all of top results, return false', () => {
+        localStorage.setItem('people', JSON.stringify(results));
+        const score = 11;
+        const maxScore = 25; // 44%
+
+        expect(peopleRanking._isHigherThanCurrentTop(score, maxScore)).toBeFalsy;
```

Comment:
Ten test i inne z `toBeFalsy` i `toBeTruthy` nic nie testują. 
Brakuje Ci wywołania asercji poprzez `()` na końcu.
Ważną zasadą jest, że jak piszemy test to najpierw trzeba napisać test (RED), który nie przejdzie. A potem dopiero ma przechodzić (GREEN), później ewentualny Refactor. Wtedy wiesz, że na pewno on coś testował i twój kod to implementuje.

Ten problem w sumie zniknie w tym przypadku jak pozbędziemy się testów na metody prywatne :)

Reply:
O, racja! Najwyraźniej wcześniej napisałam to inaczej, pewnie jakieś toEqual(false/true), ale robiłam właśnie tak, jak piszesz, a potem, jak już wszystko działało, przypomniałam sobie o tych expectach - toBeFalsy/Truthy i zmieniłam "na pałę" :D A że wszystko przechodziło, to nie zwróciłam uwagi.